templates/download/raspberry-pi/index.html

```diff
@@ -208,135 +343,233 @@ <h6 class="u-text-light">
         }}
       </div>
     </div>
+    <div class="col-2">
+      <p class="u-no-padding--top" style="margin-bottom: .5rem;">
+        Raspberry Pi Zero 2
+      </p>
+      <hr>
+      <div>
+        {{ image (
+          url="https://assets.ubuntu.com/v1/a79a3cee-raspberry-pi-z-2.png",
+          alt="",
+          width="70",
+          height="46",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <h6 class="u-text-light">
+        <em>
+          * 32-bit only
+        </em>
+      </h6>
+    <div>
   </div>
 </section>
 
-
-{# desktop latest #}
-<section class="p-strip">
-  <div class="u-fixed-width">
-    <h2>
-      Download Ubuntu Desktop
-    </h2>
+{# core #}
+<section class="p-strip--light {% if releases.latest.short_version < releases.lts.short_version %}is-bordered{% endif %}">
+  <div class="row">
+    <div class="col-6">
+      <h2 class="u-sv2">
+        Download Ubuntu Core
+      </h2>
+      <p class="p-muted-heading">
+        Recommended core
+      </p>
+    </div>
   </div>
   <div class="row">
     <div class="col-6">
       <h3>
-        Ubuntu Desktop {{ releases.latest.full_version }}
+        Ubuntu Core 20 arm64
```

Comment:
According to the copy doc:
```suggestion
        Ubuntu Core 20
```